modules/extensions-support/src/main/resources/META-INF/mule-extension-model.json

```diff
@@ -1825,6 +1825,561 @@
       ],
       "description": "Processes the nested list of message processors asynchronously using a thread pool",
       "modelProperties": {}
+    },
+    {
+      "name": "try",
+      "kind": "scope",
+      "blocking": true,
+      "executionType": "CPU_LITE",
+      "errors": [],
+      "output": {
+        "type": {
+          "format": "java",
+          "type": "Any"
+        },
+        "hasDynamicType": false,
+        "description": "",
+        "modelProperties": {}
+      },
+      "outputAttributes": {
+        "type": {
+          "format": "java",
+          "type": "Object",
+          "annotations": {
+            "typeId": "org.mule.runtime.api.message.Attributes",
+            "classInformation": {
+              "hasDefaultConstructor": false,
+              "isInterface": true,
+              "isInstantiable": false,
+              "isAbstract": true,
+              "isFinal": false,
+              "implementedInterfaces": [
+                "java.io.Serializable"
+              ],
+              "parent": "",
+              "genericTypes": []
+            },
+            "typeAlias": {
+              "value": "Attributes"
+            }
+          },
+          "fields": []
+        },
+        "hasDynamicType": false,
+        "description": "",
+        "modelProperties": {}
+      },
+      "transactional": true,
+      "requiresConnection": false,
+      "parameterGroupModels": [
+        {
+          "parameters": [
+            {
+              "type": {
```

Comment:
nice formatting... check `4.x-ce/extensions/db/target/classes/META-INF/extension-model.json`and look for `transactionalAction`